edge_computing/installing_with_siteconfig_operator/cnf-understanding-siteconfig-operator.adoc

```diff
@@ -0,0 +1,37 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cnf-understanding-siteconfig-operator"]
+= Understanding the {sco}
+include::_attributes/common-attributes.adoc[]
+:context: siteconfig-operator
+
+toc::[]
+
+With the new {sco}, you can deploy clusters with all available installation methods.
+The {sco} uses the unified `ClusterInstance` API, which decouples parameters that define a cluster from the manner in which the cluster is deployed.
+This separation removes certain limitations that are presented by the `SiteConfig` kustomize plugin (`siteconfig-v1`) in the current xref:../../edge_computing/ztp-deploying-far-edge-sites.adoc#ztp-deploying-far-edge-sites[{ztp-first} flow], such as agent cluster installations and scalability constraints posed by ArgoCD.
+Thanks to the unified `ClusterInstance` API, the {sco} offers the following improvements:
+
+Isolation:: Separates cluster definition from the installation method. The {sco} CRD, the `ClusterInstance` CR, captures the cluster definition, while the cluster templates capture the cluster architecture and installation methods.
+Unification:: The {sco} unifies both Git and non-Git workflows. Users can apply the `ClusterInstance` CR directly on the hub cluster or synchronize them through a GitOps solution, like ArgoCD.
+Consistency:: Maintains a consistent API across all installation methods, whether using the Assisted Installer, the Image Based Install Operator, or any other custom template-based approach.
+Scalability:: Achieves better scalability per cluster than `siteconfig-v1`. Configured as an ArgoCD application, `siteconfig-v1` can manage a maximum of 300 `SiteConfig` CRs, whereas the `siteconfig-v2` supports greater scalability.
```

Comment:
As it's the first time mentioning `siteconfig-v2` to describe more what it is? The siteconfig-v2 plugin, application, configuration?